src/benchmarks/micro/MicroBenchmarks.csproj

```diff
@@ -137,6 +137,12 @@
     <Compile Remove="runtime\Math\Functions\Single\**\*.cs" />
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
I don't understand why this is happening, VS keeps showing F# files in the CS solution:
![image](https://github.com/dotnet/performance/assets/5451366/5aa49f8a-a46c-4bab-91e8-9942f722013a)

This hack at least hides the files themselves but still shows the folder:
![image](https://github.com/dotnet/performance/assets/5451366/ed8e6a3a-1272-4122-a39f-97efc514c67e)

Probably some linking is happening somewhere but I don't understand where.